packages/camera/ios/Classes/CameraPlugin.m

```
@@ -176,7 +175,7 @@ - (void)captureOutput:(AVCaptureOutput *)output
     }
   }
   if (!CMSampleBufferDataIsReady(sampleBuffer)) {
-    NSLog(@"sample buffer is not ready. Skipping sample");
+      _eventSink(@{@"event" : @"error", @"errorDescription" : @"sample buffer is not ready. Skipping sample"});
```

This it the event I'm not sure we should keep.

---

If it's not an error then just let it fail silently.